enki/PhysicalEngine.cpp

```diff
@@ -225,14 +225,15 @@ namespace Enki
 	
 	const double PhysicalObject::g = 9.81;
 	
-	PhysicalObject::PhysicalObject(void) :
+	PhysicalObject::PhysicalObject(bool mouvable) :
```

Comment:
I thought that *having infinite mass* and *unmovable* were synonyms.

I think the property is ***pickable***. An object is pickable, if the viewer can add it to a set of picked objects. The set may be limited to size one, but that depends on the viewer.

There is nothing to say what the viewer chooses to do with that set. It may interpret a “mouse down–slide–mouse up” gesture as a request to move an object, but it could just as easily interpret that gesture as a request to increase the volume of the sounds from that object.

A future, more sophisticated, viewer might allow an operation like, pick some of the robots, stop their VMs, and move them all to the their starting points.